mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -7705,8 +7715,61 @@ resource "google_compute_instance" "foobar2" {
   }
 
 }
-`, instance, confidentialInstanceType, instance)
```

Comment:
Why was this line deleted?

Reply:
That line was replaced with
`, instance, confidentialInstanceType, instance, instance, confidentialInstanceType, instance) on line 7770.

Reply:
This breaks the GA build because in GA this function no longer terminates.

Reply:
My bad, the version guard <% end -%> was placed incorrectly, should be fixed now.

Reply:
Fixed with `else` cause for old arguments.